src/components/dumb/OeLoader.vue

```diff
@@ -41,12 +59,61 @@ import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
   -webkit-border-radius: 50px;
   -moz-border-radius: 50px;
   border-radius: 50px;
-}
-.spinner-container .spinner {
-  position: absolute;
-  top: 25%;
-  left: 25%;
-  font-size: 50px;
-  color: rgb(255, 255, 255);
+
+  .spinner {
+    position: absolute;
+    top: 25%;
+    left: 25%;
+    font-size: 50px;
+    color: rgb(255, 255, 255);
+  }
+
+  &--inline {
+    box-shadow: none;
+    width: 18px;
+    height: 18px;
+    background: rgb(117, 63, 127);
+    position: relative;
+    display: inline-block;
+    top: 0;
+    left: 0;
+    padding: 0;
+    margin: 0 0.5rem;
+    transform: none;
+    vertical-align: middle;
+
+    .spinner {
+      font-size: 16px;
+      top: 0;
+      left: 0;
+    }
+  }
+
+  &--small {
+    width: 14px;
```

Comment:
is het hier niet beter om te werken met vh ipv px?

Reply:
Getest op ipad en iphone met vh, em en rem, maar in alle cases doet het funky dus ik ga de pixels behouden, dat werkt wel :)